messages/btc.proto

```diff
@@ -35,6 +35,8 @@ message BTCPubRequest {
     XPUB = 1;
     YPUB = 2;
     ZPUB = 3;
+    VPUB = 5;
```

Comment:
Please put them in order of number, so the next person does not add `= 5` for the next var ;) It's a bit unfortunate but it is normal to append the new thing at the end.

Reply:
Done.